lnbits/extensions/scheduledpayments/views_api.py

```diff
@@ -0,0 +1,101 @@
+from http import HTTPStatus
+
+import crontools as ct
+from fastapi import Query
+from fastapi.params import Depends
+from loguru import logger
+from starlette.exceptions import HTTPException
+
+from lnbits.core.crud import get_user
+from lnbits.decorators import WalletTypeInfo, get_key_type, require_admin_key
+
+from . import scheduledpayments_ext
+from .crud import (
+    create_schedule,
+    delete_schedule,
+    get_events,
+    get_schedule,
+    get_schedules,
+    update_schedule,
+)
+from .models import CreateScheduleData, UpdateScheduleData
+
+
+@scheduledpayments_ext.get("/api/v1/schedule/{schedule_id}", status_code=HTTPStatus.OK)
+async def api_schedule(schedule_id: str):
```

Comment:
- this endpoint is not secured. Is this on purpose?

Reply:
No, pushed fix.